great-idea-website/css/index.css

```diff
@@ -53,14 +53,146 @@ table {
 }
 
 html, body {
-    height: 100%;
+	margin: 0 auto;
     font-family: 'Titillium Web', sans-serif;
 }
 
 h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
+	
+	margin-bottom: 15px;
+	font-family: 'Bangers', cursive;
     letter-spacing: 1px;
-    margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+/* ************************************************
+*                    CONTAINER                     *
+***************************************************/ 
```

Comment:
Super helpful comments which break up the code to manageable pieces. I appreciate this. ❤️